synapse/cortex.py

```diff
@@ -188,6 +208,49 @@ def __init__(self, dirn):
         self.addCoreMods(mods)
 
         self._initCryoLoop()
+        self._initPushLoop()
+        self._initFeedLoops()
+
+    def _initPushLoop(self):
+
+        if self.conf.get('splice:sync') is None:
+            return
+
+        self._runPushLoop()
```

Comment:
we should hold a ref to this thread and join() him on fini

Reply:
yep.  good catch.